docs/en/status-communities/vote-to-feature-a-status-community.md

```diff
@@ -0,0 +1,78 @@
+---
+id: 691
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Vote to feature a Status Community
+
+If you're passionate about a community and wish for more people to discover it, you can vote for it to be featured.
+
+All SNT holders can vote for their favourite public communities. The five top communities get a spot in the Featured section in Communities, making them more visible to everyone.
+
+## Vote to feature a Status Community
+
+Votes happen on the [Community Directory Curation dApp :octicons-tab-external-16:][curate.status.app]. Your voting power is determined by how many SNT tokens you stake. You don't spend the tokens to vote, but you must hold the tokens during validation and finalization periods. A [gas fee][understand-network-fees] in ETH is required for initiating, validating and finalizing a vote.
+
+Voting involves five steps:
+
+- [x] [Connect your wallet][connect-the-wallet].
+- [x] [Initiate a round of vote][initiate-voting].
+- [x] [Cast your vote][cast-the-vote].
+- [x] [Verify the vote][verify-the-vote].
+- [x] [Finalize the vote][finalize-the-vote].
+
+!!! note
+    Status does not charge you any fees for voting.
+
+!!! note
+    To know more about the dApp and the rules of each step, check out [About voting to change community visibility][about-voting-to-change-community-visibility].
+
+## Step 1: Connect your wallet {: #connect-the-wallet}
+
+Make sure your wallet contains enough SNT for voting and ETH for the gas fee.
+
+1. Open your wallet and switch the network to Optimism.
+1. Open the Community Directory Curation dApp and select **Connect** from the top navigation.
+1. Select **Confirm** in your wallet.
+
+!!! note
+    You can use Status Wallet or MetaMask.
+
+## Step 2: Initiate a round of vote {: #initiate-voting}
+
+After the previous round of vote is finalized, a new round can be initiated. Each round lasts one week.
+
+A round of vote is initiated when the first voter [cast a vote][cast-the-vote]. The voter must pay a gas fee.
+
+## Step 2: Cast your vote {: #cast-the-vote}
+
+The voting period lasts for 5 days. Voters don't need to pay any gas fee.
+
+1. From the top navigation, select **Directory** to view all public communities.
+1. Find the community you want to vote for and select **Feature this community!**.
+1. Choose the amount of SNT to stake and select **Confirm vote to feature communtiy**.
+1. Confirm the transaction and pay the gas fee from your wallet.
+
+## Step 3: Verify the vote {: #verify-the-vote}
```

Comment:
We should emphasize this step is optional, but the more peers participate, the better it is for vote integrity.